docs/dev_guide/ceph_s3_tests/ceph_s3_tests_pending_list_status.md

```diff
@@ -46,3 +46,7 @@ Attached a table with tests that where investigated and their status (this table
 | test_object_create_bad_date_none_aws2                     | Internal Component | [438](https://github.com/ceph/s3-tests/issues/438)                    | It used to pass in the past (not related to code change in our repo) |
 | test_bucket_create_bad_authorization_invalid_aws2         | Internal Component | [438](https://github.com/ceph/s3-tests/issues/438)                    | It used to pass in the past (not related to code change in our repo) |
 | test_bucket_create_bad_date_none_aws2                     | Internal Component | [438](https://github.com/ceph/s3-tests/issues/438)                    | It used to pass in the past (not related to code change in our repo) |
+| test_get_object_ifnonematch_good                     | Internal Component |                    | It used to pass in the past (not related to code 
```

Comment:
@shirady could you please create an issue to investigate why it's failing?

Reply:
Opened an issue in #8079